### PR TITLE
Added _config.yml and _toc.yml files to build jupyterbook

### PR DESCRIPTION
**Pull request by suppathak**
Added _config.yml and _toc.yml files to build jupyterbook

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+format: jb-book
+root: README
+parts:
+  - caption: Relevant Research Papers
```
Comment (chauhankaranraj) [inline _toc.yml:4]:
```suggestion
  - caption: Previous Work Done
```
Would it make sense to keep caption names a bit broad?

File: _toc.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    chapters:
+    - file: docs/Relevant_papers.md
+  - caption: Getting Started
+    chapters:
+    - file: notebooks/getting_started_notebook
+    - file: pyxis_mapping
```
Comment (chauhankaranraj) [inline _toc.yml:10]:
```suggestion
    - file: notebooks/pyxis_mapping
```
The notebook isn't rendering for me without this prefix

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
Left a small comment but mostly looks good :smile:

**Comment by chauhankaranraj** [comment]
Also, imo updating the notebook/md headers could be a pretty neat enhancement and might make the jupyterbook look even better (screenshots attached just to give an idea). Maybe this can be explored in a future PR?

Screenshot from 2021-11-11 17-11-09

Screenshot from 2021-11-11 17-09-35

**Comment by suppathak** [comment]
> Also, imo updating the notebook/md headers could be a pretty neat enhancement and might make the jupyterbook look even better (screenshots attached just to give an idea). Maybe this can be explored in a future PR?

@chauhankaranraj , Thanks :),  Yes that is a good idea. I also feel the need to edit some of the headers for individual notebook.

**Review by chauhankaranraj — APPROVED** [review]
Thanks for the changes, looks good! Also, going forward I'd suggest creating PRs from your fork (you'll have to re-fork coz the repo visibility was changed to public) instead of a branch on upstream :)

/approve

**Pull request merged**